darts/models/gaussian_process_filter.py

```diff
@@ -1,3 +1,8 @@
+"""
+Gaussian Processes
+------------------
```

Comment:
In some cases you removed these lines. I don't have a preference but I suppose we should be consistent

Reply:
Yes, removing the dotted lines removes the page from the generated doc.